### PR TITLE
Update docs

### LINKED ISSUE #9: Update the iOS guide
- Add all the devices you want to use in the portal
- Add a wildcard App ID
- Open keychain
  - request a certificate signing request
- Open certificates page, create a new iOS Distribution (App Store and Ad Hoc)
- Download and double click to install. This will open keychain
- Open the profiles tab on the Certs Ids and Profiles 
- Create a new “Ad Hoc” profile
- Use the app ID you created with the wildcard
- Pick the certificate that you just created probably todays date
- Add the devices that the profile will use
  - Name it “distribution”

# Trouble shooting
Delete the older Apple World Wide Developer

### PR DESCRIPTION
**Pull request by chicoxyzzy**
closes #9

### CODE REVIEW

File: build/index.html [hunk 1]
```diff
@@ -103,12 +103,12 @@ <h2>Install</h2>
...
             <h2>Getting Started</h2>
             <p>
-              In a new directory run <code>ssc --init</code>. This will create
+              In a new directory run <code>ssc init</code>. This will create
               an <code>ssc.config</code> file, and create a <code>src</code>
               directory with an a <code>index.html</code> file in it.
             </p>
             <p>
-              Running <code>ssc . -r</code> will create <code>build</code> directory
+              Running <code>ssc -r .</code> will create <code>build</code> directory
```
Comment (Raynos) [inline build/index.html:111]: `ssc compile -r .` ?

File: src/docs/ios.md [hunk 2]
```diff
@@ -17,81 +17,68 @@ and phystical `devices`.
...
+1. Add a name for your new distribution profile (we recommend to name it "distribution").
+1. Download the profile and double click it. This action will open Xcode. You can close it after it's completely loaded.
+1. Place your profile to your project directory (same directory as ssc.config). *The profiles are secret, add your profile to `.gitignore`*.
 
 ### Configuration
 
-Edit the following fields in the `ssc.config` file of your project.
+1. Set the "ios_team_id" value in `ssc.config` to the ???
+1. Set the "ios_distribution_method" value in `ssc.config` to the ???
+1. Set the "ios_signing_certificate" value in `ssc.config` to the ???
```
Comment (chicoxyzzy) [inline src/docs/ios.md:62]: I don't where to find these values

File: src/docs/ios.md [hunk 3]
```diff
@@ -17,81 +17,65 @@ and phystical `devices`.
...
 ### Configuration
 
-Edit the following fields in the `ssc.config` file of your project.
-
-```settings
-apple_teamId: Z3M838H537
-apple_debug_distribution_method: ad-hoc
-apple_debug_signing_certificate: iOS Distribution (App Store and Ad Hoc)
-apple_debug_provisioning_profile: src/development.mobileprovision
+1. Set the "ios_team_id" value in `ssc.config` to the value of Team ID [here][https://developer.apple.com/account/#!/membership/]
```
Comment (chicoxyzzy) [inline src/docs/ios.md:57]: this one is hacky, since I rely on the fact that now it redirects to the proper team

File: src/docs/ios.md [hunk 4]
```diff
@@ -17,81 +17,65 @@ and phystical `devices`.
...
+1. Set the "ios_distribution_method" value in `ssc.config` to the `ad-hoc`
+1. Set the "ios_signing_certificate" value in `ssc.config` to the ???
```
Comment (chicoxyzzy) [inline src/docs/ios.md:59]: Is the only way to find this one is to find it in the Keychain?

**Pull request merged**